### DISCUSSION TITLE
Ability to drag window above the url bar

### DISCUSSION THREAD

**Discussion by valery-kirichenko** [zen-browser/desktop#2157] [Ideas]
Currently, if the browser window is maximized, you can only drag it using the red areas. If you remove flexible areas, you have only small area above application menu button to do that. I suggest making topmost area (green) above URL bar draggable.
![image](https://github.com/user-attachments/assets/5d4ce0a6-d6c1-4f5c-9550-254b9e9f1021)

**Comment by stripedgoat** [comment]
You can drag the window by grabbing it by the sidebar

**Reply by valery-kirichenko** [reply]
That's not what I am used to. Basically every browser allows grabbing by the top 1px line (well, except for Firefox for some reason). And if (when) Zen Browser gets horizontal tab layout, this will become even more necessary.

**Reply by stripedgoat** [reply]
Adapt! 😃

**Comment by SyndicatedPillbug** [comment]
Better drag would be really nice. The sidebar drag is... fine? I guess? But there's no reason from a ux standpoint not to at least consider the option. Stripedgoat might be joking, but "adapt" is really terrible as a design methodology.
